src/platform/nrfconnect/BUILD.gn

```diff
@@ -95,6 +95,8 @@ static_library("nrfconnect") {
       "wifi/WiFiManager.cpp",
       "wifi/WiFiManager.h",
     ]
+
+    public_deps += [ "${chip_root}/src/messaging:messaging_mrp_config" ]
```

Comment:
Maybe let's move it under the `chip_enable_wifi` brackets, as it doesn't seem to be needed for Thread.

Reply:
Hmm, it is there already. Please check it one more time.

Reply:
Oh right, that's my oversight.